package.json

```diff
@@ -34,6 +34,7 @@
     "@popperjs/core": "^2.11.2",
     "angular-oauth2-oidc": "~13.0.1",
     "bootstrap": "^5.1.3",
+    "google-libphonenumber": "^3.2.27",
```

Comment:
Ta libka zajmuje 559 KB przed gzipem. Wydaje mi się to overkillem dla funkcjonalności, jaką oferuje, na frondendzie musimy uważać na finalny rozmiar JS-owych bundli...

Z readme głównej libki:
https://github.com/google/libphonenumber#third-party-ports
wynika, że jest do wyboru też port https://github.com/catamphetamine/libphonenumber-js, który zajmuje 110 KB, nie >500. Poszedłbym tą drogą.

cc @bbartosiak, potencjalnie do przegadania

Reply:
Wg [dokumentacji](https://github.com/catamphetamine/libphonenumber-js#min-vs-max-vs-mobile-vs-core) tego portu na JSa, istnieje opcja skonfigurowania tej libki tak, żeby do bundla dołączane było tylko _metadata_ dla tych krajów, które chcemy:
> Sometimes (rarely) not all countries are needed, and in those cases developers may want to [generate](https://github.com/catamphetamine/libphonenumber-js#customizing-metadata) their own "custom" metadata set. For those cases, there's libphonenumber-js/core sub-package which doesn't come pre-packaged with any default metadata set and instead accepts metadata as the last argument of each exported function.

Skoro w select'cie na froncie i tak jest do wyboru tylko kilka numerów kierunkowych, może to też jest opcja?

Reply:
No i też trzeba pamiętać, że skoro wybieramy (?) taką opcję "ograniczoną", to wciąż trzeba dobrze wyświetlać błąd walidacji numeru na backendzie - tam ta walidacja może być jednak szczelniejsza, bo nie musimy iść na kompromisy z libką.

Reply:
wycofujemy PR, nie wchodzimy z walidacjami na froncie z tą biblioteką.

Reply:
@sfc-gh-pnescior podejrzewam, że i po ograniczeniu się do paru krajów ta libka będzie duża, bo ona taka duża po prostu nie powinna być tylko ze względu na zawartość, ale raczej ze względu na:
> browserify-compatible wrapper around the original unmodified library installable via npm, **which packs the Google Closure library**